x-pack/legacy/plugins/ml/common/types/common.ts

```diff
@@ -8,6 +8,31 @@ export interface Dictionary<TValue> {
   [id: string]: TValue;
 }
 
+export interface StatsBarStat {
```

Comment:
So far we had only some very very genreric types in here. I think the types added here should go into their own file, they are quite specific.